### PR TITLE
[BUG] Remove unnecessary durations from cnt.py

### PR DESCRIPTION
**Pull request by withmywoessner**
#### Reference issue
Fixes: [#11824](https://github.com/mne-tools/mne-python/issues/11824)

#### What does this implement/fix?
Removes unnecessary durations from the annotations/events after reading cnt files.

### CODE REVIEW

File: mne/io/cnt/cnt.py [hunk 1]
```diff
@@ -145,9 +145,7 @@ def _update_bad_span_onset(accept_reject, onset, duration, description):
             event_type=type(my_events[0]),
             data_format=data_format,
         )
-        duration = np.array(
-            [getattr(e, "Latency", 0.0) for e in my_events], dtype=float
-        )
+        duration = np.zeros_like(my_events).astype(float)
```
Comment (hoechenberger) [inline mne/io/cnt/cnt.py:148]:
```suggestion
        duration = np.zeros_like(my_events, dtype=float)
```

File: mne/io/cnt/cnt.py [hunk 2]
```diff
@@ -145,9 +145,7 @@ def _update_bad_span_onset(accept_reject, onset, duration, description):
             event_type=type(my_events[0]),
             data_format=data_format,
         )
-        duration = np.array(
-            [getattr(e, "Latency", 0.0) for e in my_events], dtype=float
-        )
+        duration = np.zeros_like(my_events, dtype=float)
```
Comment (hoechenberger) [inline mne/io/cnt/cnt.py:148]: I just figured, according to our docs, we can simply pass a float here and all annotations will have the same duration. Hence this should workL

```suggestion
        duration = 0.
```

Comment (hoechenberger) [reply mne/io/cnt/cnt.py:148]: But I'm not sure if `_update_bad_span_onset()` can handle this

Comment (withmywoessner) [reply mne/io/cnt/cnt.py:148]: Yeah, I made that. Should I change to accept a float or just leave it how it is?

File: doc/changes/latest.inc [hunk 3]
```diff
@@ -37,6 +37,7 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix bug where :func:`mne.io.read_raw_cnt` imports unnecessary durations (:gh:`11828` by `Jacob Woessner`)
```
Comment (agramfort) [inline doc/changes/latest.inc:40]: are you a new contributor @withmywoessner ? if not can you put your name below the new contrib or if so can you put the newcontrib tag as the line below?

**Review by agramfort — COMMENTED** [review]
now is this a bug in the file writer? it inserts a dummy place older in the file and it's not actually an aquisition parameter?

**Comment by withmywoessner** [comment]
> now is this a bug in the file writer? it inserts a dummy place older in the file and it's not actually an aquisition parameter?

Hey @agramfort. From investigating the neuroscan software and the documentation [this reader references ](http://paulbourke.net/dataformats/eeg/), I believe that that parameter doesn't do anything as neuroscan represents each event using a separate event code. There are technically annotations (used for representing bad spans; [I already wrote something to account for this earlier](https://github.com/mne-tools/mne-python/blob/68e7272563fa88c2f0bd6d3e551eac9c51c2d07d/mne/io/cnt/cnt.py#L76C4-L115C10) ) that have a duration, but in neuroscan these are represented by two event codes specifying the start and the end, they do not use `'Latency'` ever.

File: doc/changes/latest.inc (hunk 3 above)
Comment (withmywoessner) [reply doc/changes/latest.inc:40]: I have already contributed before.

Comment (larsoner) [reply doc/changes/latest.inc:40]: @withmywoessner don't forget the first part of the conditional above:

> if not can you put your name below the new contrib

We try to have the `:newcontrib:` at the top of each section, so this line needs to drop down a few lines

File: mne/io/cnt/cnt.py [hunk 4]
```diff
@@ -145,9 +145,7 @@ def _update_bad_span_onset(accept_reject, onset, duration, description):
             event_type=type(my_events[0]),
             data_format=data_format,
         )
-        duration = np.array(
-            [getattr(e, "Latency", 0.0) for e in my_events], dtype=float
-        )
+        duration = np.zeros(len(my_events), dtype=float)
```
Comment (larsoner) [inline mne/io/cnt/cnt.py:148]:
```suggestion
        # There is a Latency field but it's not useful for durations, see
        # https://github.com/mne-tools/mne-python/pull/11828
        duration = np.zeros(len(my_events), dtype=float)
```

**Review by larsoner — COMMENTED** [review]
Thanks for the thorough investigation @withmywoessner ! Looks like it's almost good to go, I'm just going to make two small adjustments then mark for merge. (Me doing it has the advantage that CircleCI won't complain / `block-unregistered-user` so things should end up totally green!)

**Comment by withmywoessner** [comment]
> name crossref is failing, make sure you're added to `doc/changes/names.inc` @withmywoessner

I thought I was:
https://github.com/mne-tools/mne-python/blob/68e7272563fa88c2f0bd6d3e551eac9c51c2d07d/doc/changes/names.inc#L215

**Comment by drammock** [comment]
> > name crossref is failing, make sure you're added to `doc/changes/names.inc` @withmywoessner
> 
> I thought I was:
> 
> https://github.com/mne-tools/mne-python/blob/68e7272563fa88c2f0bd6d3e551eac9c51c2d07d/doc/changes/names.inc#L215

looks like you figured it out already, but for posterity:  

```rst
`Jacob Woessner` is interpreted as an API link, whereas `Jacob Woessner`_ is interpreted as a cross-reference (note the underscore).  The exception is when using the "newcontrib" role: like all roles, it should look like :role:`content` (no trailing underscore), in other words :newcontrib:`Jacob Woessner`
```

**Pull request merged**